### PR TITLE
[GRPH-86] fix duplicate ops returned in get_account_history

### PR DESCRIPTION
**Pull request by srpatel19590**
## Bug Description
Call the cli_wallet api with the params:
```{"id":"5665571840","jsonrpc":"2.0","method":"get_account_history","params":["alanguanwz9",1000]}```
The return result contains duplicate id : 1.11.323175501, 1.11.320758270

## To Run Test:
`./tests/cli_test -t account_history_pagination`

### CODE REVIEW

**Comment by srpatel19590** [comment]
@bobinson I will resolve conflicts here once you merge GRPH-3

**Comment by oxarbitrage** [comment]
please rebase on `develop` or open new pull request for this. thank you and sorry about that.

**Comment by srpatel19590** [comment]
@oxarbitrage No problem. PR is updated.

**Comment by srpatel19590** [comment]
This PR needs to be merged after feature/GRPH-3. However, I have added account_history plugin to fix the test

File: tests/cli/main.cpp [hunk 1]
```diff
@@ -438,3 +440,42 @@ BOOST_FIXTURE_TEST_CASE( cli_vote_for_2_witnesses, cli_fixture )
       throw;
    }
 }
+
+///////////////////////
+// Check account history pagination (see peerplay-core/issue/1176)
```
Comment (oxarbitrage) [inline tests/cli/main.cpp]: `(see peerplay-core/issue/1176)` should be removed or replaced with `bitshares-core`. Thanks.

**Comment by oxarbitrage** [comment]
https://github.com/bitshares/bitshares-core/pull/1179

File: tests/cli/main.cpp (hunk 1 above)
Comment (srpatel19590) [reply tests/cli/main.cpp]: It is removed now.

**Pull request merged**